src/com/goide/actions/tool/GoTypeFileAction.java

```diff
@@ -38,6 +40,11 @@ protected boolean isAvailableOnFile(VirtualFile file) {
   protected GoExecutor createExecutor(@NotNull Project project, @Nullable Module module, @NotNull String title, @NotNull String filePath) {
     VirtualFile executable = getExecutable(project, module);
     assert executable != null;
-    return GoExecutor.in(project, module).withExePath(executable.getPath()).withParameters("-e", "-a", "-v", filePath).showOutputOnError();
+    File file = new File(filePath);
```

Comment:
Here is convertation virtualFile -> filePath -> File that looks lame. Please extract another createExecutor method in base class with VirtualFile parameter instead of filepath